### ISSUE TITLE
After I upgraded "@nuxthub/core": "^0.5.14" to "^0.5.16" the following problem occurred

### ISSUE THREAD

**Issue by yangzhaokun** [nuxt-hub/core#95]
repo:nuxt-todos-edge
After I upgraded "@nuxthub/core": "^0.5.14" to "^0.5.16" the following problem occurred
I tried to downgrade to 0.5.14, the WARN is still there

[nuxt:tailwindcss 00:41:38] WARN You have provided functional plugins in tailwindcss.config in your Nuxt configuration that cannot be serialized for Tailwind Config. Please use tailwind.config or a separate file (specifying in tailwindcss.configPath) to enable it with additional support for IntelliSense and HMR.

![image](https://github.com/nuxt-hub/core/assets/12760701/9b53475d-b7a6-45bc-9100-1962a5e9f4ec)

**Comment by yangzhaokun** [comment]
I have found that the "pnpm update" command should not be used, otherwise it will definitely cause this result.
only upgrading core to 0.5.16  won't be a problem.

![image](https://github.com/nuxt-hub/core/assets/12760701/7d35175c-8f59-4ffb-a383-01dc9ec5d453)

**Comment by yangzhaokun** [comment]
I have found the source of the error, so as not to mislead others, I will write down the specific reasons here

When running npm run dev, this module will look for these files:
`./assets/css/tailwind.css`
`./tailwind.config.{js,cjs,mjs,ts}`
If these files don't exist, the module will automatically generate a basic configuration for them, so you don't have to create these files manually.
You can create the tailwind.config.js file by running the following command:
`npx tailwindcss init`

If you don't want to inject the CSS file, you can set cssPath to false , in nuxt.config.ts
```
export default defineNuxtConfig({
  tailwindcss: {
    cssPath: false,
  }
})

```

more info : https://tailwindcss.nuxtjs.org/getting-started/configuration

**Comment by jits** [comment]
@yangzhaokun — https://github.com/nuxt/ui/pull/1665 should fix this warning message (which is included in the just released  v2.16.0 of Nuxt UI).
